translations/cy/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,526 @@
+# Welsh translations for PROJECT.
+# Copyright (C) 2022 ORGANIZATION
+# This file is distributed under the same license as the PROJECT project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2022-10-21 16:52+0100\n"
+"PO-Revision-Date: 2022-10-21 16:48+0100\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: cy\n"
+"Language-Team: cy <LL@li.org>\n"
+"Plural-Forms: nplurals=5; plural=(n==1 ? 1 : n==2 ? 2 : n==3 ? 3 : n==6 ?"
+" 4 : 0);\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.10.3\n"
+
+#: app.py:121
+msgid "Apply for funding to save an asset in your community"
+msgstr "Cais am gyllid i achub ased yn eich cymuned"
+
+#: frontend/templates/footer.html:9
+msgid "Privacy"
+msgstr "Preifatrwydd"
+
+#: frontend/templates/footer.html:14
+msgid "Cookies"
+msgstr "Cwcis"
+
+#: frontend/templates/footer.html:19
+msgid "Accessibility Statement"
+msgstr "Datganiad Hygyrchedd"
+
+#: frontend/templates/footer.html:24
+msgid "Contact us"
+msgstr "Cysylltwch â ni"
+
+#: frontend/templates/footer.html:33
+msgid "All content is available under the"
+msgstr "Mae'r holl gynnwys ar gael o dan y"
+
+#: frontend/templates/footer.html:34
+msgid "Open Government Licence v3.0"
+msgstr "Trwydded Llywodraeth Agored v3.0"
+
+#: frontend/templates/footer.html:34
+msgid "except where otherwise stated"
+msgstr "ac eithrio lle nodir yn wahanol"
+
+#: frontend/templates/header.html:37
+msgid "This is a new service - your"
+msgstr "Mae hwn yn wasanaeth newydd - bydd eich"
+
+#: frontend/templates/header.html:39
+msgid "feedback"
+msgstr "adborth"
+
+#: frontend/templates/header.html:39
+msgid "will help us to improve it."
+msgstr "yn ein helpu i'w wella."
+
+#: venv/Lib/site-packages/click/_termui_impl.py:496
```

Comment:
We should not be pulling through localisation strings from libraries.
Please can we add a `babel.cfg` to exclude these, as in https://github.com/communitiesuk/funding-service-design-frontend/pull/179

Reply:
will do